build/generate.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 ./vendor/k8s.io/code-generator/generate-groups.sh \
 	all \
-	github.com/MarSik/kubevirt-ssp-operator/pkg/client \
-	github.com/MarSik/kubevirt-ssp-operator/pkg/apis \
+	github.com/kubevirt/kubevirt-ssp-operator/pkg/client \
```

Comment:
Can we change the package path to `kubevirt.io/kubevirt-ssp-operator/...`, like other kubevirt components have?

Reply:
I see that some components use the github path. So this is probably ok.

Reply:
Yea, I noticed all other components except for kubevirt are prefixed with `github.com` and not `kubevirt.io`, maybe we can change that in the future